_common/build_all/CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ endmacro()
 
 arcane_set_if_unset(ARCANE_WANT_NUMERICS TRUE BOOL "Compile numerical library")
 arcane_set_if_unset(ARCANE_WANT_TOTALVIEW TRUE BOOL "Compile total view display")
-arcane_set_if_unset(ARCANE_DEFAULT_PARTITIONER Metis STRING "Default partitioner")
+arcane_set_if_unset(ARCANE_DEFAULT_PARTITIONER Auto STRING "Default partitioner")
```

Comment:
Perhaps list all possibilities here ? If `arcane_set_if_unset` behaves like `option` it allows to have help with CMake GUI.

Reply:
This file is only used when you compile all the framework (arccon+axlstar+arccore+arcane) in one pass.
I will add the `option` in the `arcane/CMakeLists.txt` file.